OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,422 @@
+# Oracle WebCenter Sites 12c R2 (12.2.1.3.0) on Docker
+*This page is best viewed on Mozilla Firefox and Google Chrome browsers.*
+
+## Contents
+
+### 1. [Introduction](#1-introduction-1)
+### 2. [Hardware and Software Requirements](#2-hardware-and-software-requirements-1) 
+### 3. [Prerequisites](#3-prerequisites-1)
+### 4. [Downloading Docker Images and Oracle WebCenter Sites Binary](#4-downloading-docker-images-and-oracle-webcenter-sites-binary-1)
+### 5. [Building Oracle WebCenter Sites Docker Images](#5-building-oracle-webcenter-sites-docker-images-1)
+### 6. [Preparing to run Oracle WebCenter Sites Docker Image](#6-preparing-to-run-oracle-webcenter-sites-docker-image-1)
+### 7. [Running Oracle WebCenter Sites Docker Image](#7-running-oracle-webcenter-sites-docker-image-1)
+### 8. [FAQs](#8-faqs-1)
+### 9. [Copyright](#9-copyright-1)
+
+## 1. Introduction
+To create web content management solutions, developers need a lightweight environment. Docker images need minimum resources, thereby allowing developers to quickly create development environments.
+
+This project offers scripts to build an Oracle WebCenter Sites image based on 12c R2 (12.2.1.3). Use this Docker configuration to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Sites, see the [Oracle WebCenter Sites Online Documentation](https://docs.oracle.com/middleware/12213/wcs/index.html).
+
+This project creates Oracle WebCenter Sites Docker image with a single node targeted for development and excludes components such as SatelliteServer, SiteCapture, and VisitorServices. This image is not supported or recommended to use on a production system.
+
+## 2. Hardware and Software Requirements
+Oracle WebCenter Sites on Docker is certified to run on hardware and software listed below:
```

Comment:
If Oracle WebCenter Sites is certified, why is it not supported or recommended in production?

Reply:
When we say production our product supports cluster. As of now Docker image only creates a single node instance. So its for development instance. We might have plans to work on kubernetes to support cluster then we will certify the scripts for production instance.

Reply:
Please don't use the word "certified" then. Change this to something like "Oracle WebCenter Sites has been tested and is known to run on the following hardware and software:"